packages/date-helpers/package.json

```diff
@@ -18,13 +18,13 @@
     "@gc-digital-talent/graphql": "workspace:*",
     "@gc-digital-talent/i18n": "workspace:*",
     "date-fns": "^3.6.0",
-    "date-fns-tz": "^3.0.0",
+    "date-fns-tz": "3.0.0",
```

Comment:
Did you intend to remove the caret here?

Reply:
Yeah, they released a breaking change in `3.0.1` :-1: